bin/docbookrx

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env ruby
 
-if File.exist?(app_lib_path = File.join(File.dirname(__FILE__), '../lib/docbookrx'))
+
+if File.exist?(app_lib_path = File.expand_path(File.join(File.dirname(__FILE__), '../lib/docbookrx')))
```

Comment:
I think this line has to be the way we had it based on the research/testing that @mogztter did for Asciidoctor core. See https://github.com/asciidoctor/asciidoctor/blob/master/bin/asciidoctor#L5-L9. Or perhaps expand_path is harmless. Is there are reason you are expanding the path?